packages/@okta/vuepress-site/docs/concepts/iam-overview-identity-management-factors/index.md

```diff
@@ -139,13 +139,13 @@ A typical user profile contains attributes such as a user's first name, last nam
 
 Okta supports four types of user profiles:
 
-- **Okta user profiles** contain information about individual end users. They have thirty-one default base attributes, and you can add custom attributes.
+- **Okta user profiles** contain information about individual end users. They have include default base attributes, and you can add custom attributes.
```

Comment:
Not sure what this is supposed to be. "They include default base attributes" perhaps?